Azure/azure-cli
# [AppService] `az functionapp create`: Use app insights connection string instead of instrumentation key

**Related command**
`az functionapp create`

**Description**
Resolves #27607 

**Testing Guide**
Create function app and check that app setting added is `APPLICATIONINSIGHTS_CONNECTION_STRING` instead of `APPINSIGHTS_INSTRUMENTATION_KEY`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
